src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
> Will that be enough to silence the warning?

Yep. That works.

> cast(Statement)-1 isn't valid D.

Is there a sentinel value I can use to assert before the `return s` statement to avoid having to use clang as an error checking tool, and also avoid jumping into undefined memory?